Rakefile

```diff
@@ -31,7 +31,8 @@ RSpec::Core::RakeTask.new(:spec) do |t|
   t.exclude_pattern = 'spec/acceptance/**/*_spec.rb'
 end
 
-desc 'Run acceptance tests'
-task :acceptance do
-  Rake::Task['litmus:acceptance:localhost'].invoke
+Cucumber::Rake::Task.new(:features) do |t|
+  t.cucumber_opts = "--format pretty" # Any valid command line option can go here.
 end
+
+task default: [:spec, :features]
```

Comment:
Note there is no rubocop because it's not defined, despite including `rubocop/rake_task` on line 2. Ideally that should actually be there.